addons/website/static/src/components/edit_head_body_dialog/edit_head_body_dialog.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="website.EditHeadBodyDialog">
+    <Dialog title="'Edit Head and Body Code'">
+        <div>Enter code that will be added into the &lt;head&gt; of every page of your site.</div>
+        <CodeEditor mode="xml" theme="'monokai'"
```

Comment:
I think this should be `mode="'xml'"` not `mode="xml"`

Reply:
KI guess it was noticed with https://github.com/odoo/odoo/pull/148063/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-1e62b5d92d202f35c4233e53cdaf45af9aad79562fed319ec511666fb0f3ad02R13?

Could you backport it?